### ISSUE TITLE
Feature request: format checking in playlist stream url

### ISSUE THREAD

**Issue by HassuX** [rameplayerorg/rameplayer-defects#38]
You can input about anything to URL field and it's accepted, there should be some kind of format checking on that field.

**Comment by fabled** [comment]
Not sure how much can be checked. The rameutil:// urils can have additional parameters, as well as the rtmp:// urls can have space delimited parameters for ffmpeg. Maybe initial proto:// could be checked, but that's about all.

**Comment by HassuX** [comment]
I agree that it's not easy task to do that checking, but at least proto:// should be checked (maybe we should only allow http/https/rtmp ?)
